commands_test.go

```diff
@@ -3564,7 +3564,7 @@ var _ = Describe("Commands", func() {
 				res, err := client.XReadGroup(&redis.XReadGroupArgs{
 					Group:    "group",
 					Consumer: "consumer",
-					Streams:  []string{"stream", ">"},
+					Streams:  map[string]string{"stream": ">"},
```

Comment:
AFAIR I've chosen `[]string{"stream", ">"}` over `map[string]string{"stream": ">"}`, because maps are unordered.

Reply:
Could we at least add some doc examples because the current API is not very intuitive?

Reply:
Or perhaps having a separate ordered field for the IDs might be most intuitive